applications.yaml

```diff
@@ -1498,18 +1468,11 @@
     id: zen.exe
     matching_strategy: Equals
   options:
-  - object_name_change
-  - tray_and_multi_window
+  - force
```

Comment:
Zen browser doesn't need force

Reply:
Zen cannot be manipulated/registered in the stack without this option, I currently start Komorebi from `komorebic enable-autostart`, and Komorebi is almost stock. Would sharing my config help debug this difference between our machines?